### DISCUSSION TITLE
How to discover & add a device previously ignored

### DISCUSSION THREAD

**Discussion by joonhshin** [andrew-codechimp/HA-Battery-Notes#2226] [Q&A] [answered]
Device ID: RCHTSENSOR

Is it possible to **un**-ignore a device so it can be added?

I accidentally ignored a device. So I went to the integrations UI to manually add it, only to find the device is not in the Add a Device device drop-down. The device is a temp & humidity sensor and I have several identical devices that's already been added.

Thanks!

**Comment by andrew-codechimp** [comment, marked as answer]
You can show ignored integrations and unignore via the integrations screen drop down

![CleanShot 2024-10-17 at 08 27 14](https://github.com/user-attachments/assets/a686d04e-2a5e-4e2d-b005-07bfe623baec)

**Reply by joonhshin** [reply]
Thanks @andrew-codechimp I was able to un-ignore the device. 

FYI, the un-ignored device didn't show up right away in the drop-down. I restarted the HA instance, the device got auto-discovered again, and I was able to configure Battery Notes.
